src/terrasoApi/schema.graphql

```diff
@@ -0,0 +1,939 @@
+schema {

```

Comment:
Question: Was this file generated automatically? If it wasn't is there a way to split it so that we can organize better, maybe even using the apps categorization that we have in the backend

Reply:
yes, the code which generates it is in the backend and was merged here: https://github.com/techmatters/terraso-backend/pull/479, although this particular schema is generated by the backend code in https://github.com/techmatters/terraso-backend/pull/481

i'm not sure the best way to handle this, if we should have the file checked into both repos, or include the one from the backend repo with a git reference in package.json or a git submodule or some such. i defaulted to checking it in because it was the most straightforward. i also don't know if it could be split up, but the code to do so would be in the backend and i think should be a future PR, but i can file an issue in the backend if that seems worth putting the effort in

Reply:
So, if there is another client that needs this file, what is the process to get it and maintain it? I don't see a version referrence in this file, how can you identify if it has changed? or if the changes here are up to date?

Reply:
re: if there's another client that needs the file, the eventual plan is for the file to live in the shared client repo

re: version reference, this is a problem but i think it was already a problem, if you make a breaking backend change there was no way before to automatically know that you need to update the frontend. and if it's a non-breaking backend change then the schemas should be compatible and so it's ok if the schema falls out of date. if you try to query a new field while the schema is out of date the types will complain and you can update the schema

it might be easier to figure out the best way to handle this by doing it for a bit. manually updating it seems annoying but at the same time the only reason you would need to manually update it is because you need to make changes to the client anyway so maybe it's not that big a deal?

Reply:
ok i think i have a better solve, see https://github.com/techmatters/terraso-web-client/pull/934#issuecomment-1552260099 for details